CONTRIBUTING.md

```diff
@@ -51,6 +53,24 @@ following terms in the code, messages, or comments. Note the alternatives we wil
    * controller, host / device, worker, proxy
    * director / performer
 
+#### Git Commits
+
+##### Rules for Git Commit Message Style
+- Separate subject from the body with a blank line
+- Use past tense verbs
+- Wrap the lines at 72 characters
```

Comment:
Same here, based on what I remember our format was, the summary line should have a limit of 60 chars, while the detailed description lines can have a limit of 70 chars

Reply:
Apparently 72 is the hard limit, so lets leave it at that